nested-passthrough/ansible/setup_host.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Set up RHOSO/RHELAI passthrough
+  hosts: "{{ override_hosts | default('all', true) }}"
+  strategy: linear
+  gather_facts: "{{ gather_facts | default(false) }}"
+  any_errors_fatal: true
+  tasks:
+    - ansible.builtin.import_role:
+        name: setup_gpu
+        tasks_from: install.yml
+    - ansible.builtin.import_role:
+        name: setup_gpu
+        tasks_from: iommu.yml
+    - ansible.builtin.import_role:
+        name: setup_gpu
+        tasks_from: gpuinfo.yml
+    - ansible.builtin.import_role:
+        name: setup_gpu
+        tasks_from: drivers.yml
```

Comment:
nit: maybe a better name would have included the word blacklist to make it more clear what it does.